tee-worker/omni-executor/aa-contracts/foundry.toml

```diff
@@ -3,9 +3,13 @@ src = "src"
 out = "out"
 libs = ["lib", "../forge-libs"]
 solc_version = "0.8.28"
+evm_version = "cancun"
 optimizer = true
 optimizer_runs = 1000000
 via_ir = true
+# Deterministic bytecode settings for CREATE2 deployments
+bytecode_hash = "none"
+cbor_metadata = false
```

Comment:
Have you tried to deploy contracts with create2 factory? I'm not sure if would cause any verification problem (or you'll need manually verify that) as etherscan probably expects some metadatas to verify the contract